.travis.yml

```diff
@@ -65,5 +61,6 @@ script:
     fi
   - | # Run tests
     if [ "${MAKE_TEST_TARGET}" != "" ]; then
-      travis_retry docker exec -t -u $(id -u $USER):$(id -g $USER) "${DOCKER_INSTANCE}" make "${MAKE_TEST_TARGET}" ${MAKE_TEST_FLAGS}
+      travis_retry docker exec -t "${DOCKER_INSTANCE}" make go_packages;
```

Comment:
Do we use Golang for everything?

Reply:
No, not really, but this is one area I know that tends to be flaky (because `go get` needs to clone some GitHub repos), and it's relatively small so we aren't wasting too much time on jobs that don't need `go_packages`.

Reply:
On a second thought, I moved this line to `install` and added a comment.